src/Microsoft.PowerApps.TestEngine/Config/TestSettings.cs

```diff
@@ -22,6 +20,11 @@ public class TestSettings
         /// </summary>
         public List<BrowserConfiguration> BrowserConfigurations { get; set; }
 
+        /// <summary>
+        /// Gets or sets the locale for the test suite being run
+        /// </summary>
+        public string Locale { get; set; } = string.Empty;
```

Comment:
I've never used `string.Empty` as the default before. I'm wondering will it cause any problem if the test plan doesn't have a `locale `field? If no, we are good to go.

Reply:
This is supported and handled in the `GetLocaleFromTestSettings` function. That is why this field is deliberately of `string` type and not `CultureInfo` in this class.